### PR TITLE
feat(admin-gui): bump kong-manager to v3.8.0

### PR DESCRIPTION
**Pull request by sumimakito**
### Summary

This pull request updates `KONG_MANAGER` to `v3.8.0`

### Checklist

- [ ] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)

### CODE REVIEW

**Review by raoxiaoyan — CHANGES_REQUESTED** [review]
Update the base to release/3.8.x when it is available

File: changelog/unreleased/kong-manager/a11y-improvements.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+message: Improved accessibility in Kong Manager.
```
Comment (raoxiaoyan) [inline changelog/unreleased/kong-manager/a11y-improvements.yml:1]: This seems very general. Do we need it?

Comment (raoxiaoyan) [reply changelog/unreleased/kong-manager/a11y-improvements.yml:1]: Perhaps, It would be better to provide more details.

Comment (Leopoldthecoder) [reply changelog/unreleased/kong-manager/a11y-improvements.yml:1]: It's not a problem to have general changelog entries. This one specifically comprises improvements in several aspects: adding `aria-*` labels to indicate different statuses of an element, supporting keyboard navigation and events in more components, etc. Listing them all will make it too long so I think it's acceptable to leave it as is.

**Review by raoxiaoyan — APPROVED** [review]

File: .requirements [hunk 2]
```diff
@@ -21,7 +21,7 @@ LUA_RESTY_WEBSOCKET=966c69c39f03029b9b42ec0f8e55aaed7d6eebc0 # 0.4.0.1
 ATC_ROUTER=ffd11db657115769bf94f0c4f915f98300bc26b6 # 1.6.2
 SNAPPY=23b3286820105438c5dbb9bc22f1bb85c5812c8a # 1.2.0
 
-KONG_MANAGER=nightly
+KONG_MANAGER=v3.8.0.0
```
Comment (AndyZhang0707) [inline .requirements:24]: Should the version be 3.8.0 for CE?

Comment (Leopoldthecoder) [reply .requirements:24]: No, `kong-manager` always uses 4-digit versions:

https://github.com/Kong/kong/blob/e89feb5b85286394330968054c02ac851fd2b781/.requirements#L18
https://github.com/Kong/kong/blob/4e006aa85d7de34668ad40d601e275b575f0d2aa/.requirements#L15

**Review by AndyZhang0707 — APPROVED** [review]

**Pull request merged**